modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -253,6 +260,35 @@ public void testDeactivate() throws Exception {
         assertFalse(ignite.cluster().active());
     }
 
+    /**
+     * Test the deactivation command with the output of the cluster name
+     * in the confirmation message.
+     *
+     * @throws Exception If failed.
+     * */
+    @Test
+    @WithSystemProperty(key = IGNITE_CLUSTER_NAME, value = CLUSTER_NAME)
+    public void testDeactivateWithClusterNameInPromptMessage() throws Exception {
+        IgniteEx igniteEx = startGrid(0);
+        assertFalse(igniteEx.cluster().active());
+
+        igniteEx.cluster().active(true);
+        assertTrue(igniteEx.cluster().active());
+
+        autoConfirmation = false;
```

Comment:
Should we reset `autoConfirmation` to default after test?

Reply:
This happens automatically when you run each test.
protected boolean autoConfirmation = true;